classes/submission/reviewer/form/ReviewerReviewStep3Form.inc.php

```diff
@@ -220,7 +220,8 @@ function execute($request) {
 		}
 
 		// Set review to next step.
-		$this->updateReviewStepAndSaveSubmission($this->getReviewerSubmission());
+		$reviewerSubmission = $this->getReviewerSubmission();
+		$this->updateReviewStepAndSaveSubmission($reviewerSubmission);
```

Comment:
This is for a reference warning, right? Maybe better to remove the `&` from the function definition for `updateReviewStepAndSaveSubmission`. The reference isn't necessary and we're trying to remove useless references throughout (gradually, as there are lots).